src/Nethermind/Ethereum.VM.Test/AbiTests.cs

```diff
@@ -29,7 +29,7 @@ internal class AbiTests
     {
         private static readonly Dictionary<string, AbiType> TypesByName = new Dictionary<string, AbiType>
         {
-            {"uint256", AbiType.UInt},
+            {"uint256", AbiType.UInt256},
```

Comment:
UInt in Solidity is uint256, that was correct, it is not a C# uint

Reply:
Yes.

This was more of a rename UInt->UInt256 as I needed also other UInt Abi types, UInt16 for example.